Public/Src/FrontEnd/Script/Evaluator/CommonConstants.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics.ContractsLight;
+using BuildXL.Utilities;
+
+namespace BuildXL.FrontEnd.Script.Evaluator
+{
+    /// <summary>
+    /// CommonConstants for value AST.
+    /// </summary>
+    public sealed class CommonConstants
+    {
+        /// <nodoc />
+        public SymbolAtom Qualifier { get; }
+
+        /// <nodoc />
+        public SymbolAtom Length { get; }
+
+        /// <nodoc />
+        private readonly StringTable m_stringTable;

```

Comment:
Very nit: move it on the top:)

Reply:
Will do no next PR

---
In reply to: [274539264](https://github.com/Microsoft/BuildXL/pull/127#discussion_r274539264) [](ancestors = 274539264)